aws/sdk/examples/apigatewaymanagement/README.md

```diff
@@ -0,0 +1,17 @@
+# AWS SDK for Rust code examples for API Gateway
+
+The Amazon API Gateway Management API allows you to directly manage runtime aspects of your deployed APIs. To use it, you must explicitly set the SDK's endpoint to point to the endpoint of your deployed API. The endpoint will be of the form https://`[api-id].execute-api.[region].amazonaws.com/[stage]`, or will be the endpoint corresponding to your API's custom domain and base path, if applicable.
```

Comment:
When we write docs like this, should we be following the convention from the dev guide around defining the parts of `[api-id].execute-api.[region].amazonaws.com/[stage]` that get replaced?

e.g. "`[api-id]` where api-id is the blah blah blah"